demos/tests/cases.py

```diff
@@ -627,7 +627,9 @@ def single_option_cases(key, *args):
                     ModelArg('deeplabv3'),
                     ModelArg('ocrnet-hrnet-w48-paddle'),
                     ModelArg('pspnet-pytorch'),
-                    ModelArg('drn-d-38'))),
+                    ModelArg('drn-d-38'),
+                    # ModelArg('erfnet'),  # TODO add after CI is up
```

Comment:
you can not comment it, because it does not have effect, everything for running model already in repo, so manually test should pass

Reply:
I did not modify this place. Do I need to adjust it?@Wovchena

Reply:
Yes, that was me. It can be uncommented. If you will, please uncomment. But if you don't do it, it's also fine, I don't want to make you resolve things after me.